velox/functions/prestosql/aggregates/tests/MinMaxTest.cpp

```diff
@@ -236,4 +236,50 @@ TEST_F(MinMaxTest, initialValue) {
   testAggregations({row}, {}, {"max(c1)"}, "SELECT -1");
 }
 
+/// TODO: Add vectorfuzzer support for decimals to use existing
+/// MinMaxTest::doTest() call.
+/// https://github.com/facebookincubator/velox/issues/3756
+TEST_F(MinMaxTest, decimalMinMax) {
```

Comment:
Perhaps, split this test into 2: shortDecimal and longDecimal. Drop MinMax suffix as it is implied by the test class name.